client/src/test/java/io/pravega/client/stream/impl/TransactionalEventStreamWriterTest.java

```diff
@@ -125,19 +127,26 @@ public void testGetTxn() throws TxnFailedException {
         Mockito.when(streamFactory.createOutputStreamForSegment(eq(segment), any(), any(), any())).thenReturn(bad);
 
         // Create a transactional event Writer
-        @Cleanup
         TransactionalEventStreamWriter<String> writer = new TransactionalEventStreamWriterImpl<>(stream, "id", controller, streamFactory, serializer,
                 config, executorService());
         writer.beginTxn();
+        writer.close();
 
-        Transaction<String> txn = writer.getTxn(txnId);
+        //Create a new transactional eventWriter
+        @Cleanup
+        TransactionalEventStreamWriter<String> writer1 = new TransactionalEventStreamWriterImpl<>(stream, "id", controller, streamFactory, serializer,
+                                                                                                  EventWriterConfig.builder().transactionTimeoutTime(10000).build(),
```

Comment:
The assumption here is that the `transactionTimeout ` passed here is the same value that was passed during the transaction creation.